yarn-project/simulator/src/client/db_oracle.ts

```diff
@@ -197,7 +198,7 @@ export interface DBOracle extends CommitmentsDB {
 
   /**
    * Returns the tagging secret for a given sender and recipient pair. For this to work, the ivpsk_m of the sender must be known.
-   * Includes the last known index used for tagging with this secret.
+   * Includes the last seen index used for tagging with this secret, or 0 if this combination hasn't been seen before.
```

Comment:
Addressing https://github.com/AztecProtocol/aztec-packages/pull/9468#discussion_r1828411694

@nventuro

Reply:
`simulator_oracle/index.ts` now reads that this is the next index to be used

Reply:
Yes, because it is true. Anyhow this is getting reworked in favor of `...asSender` and `...asRecipient` variants of the methods in an upcoming PR that separates the indexes to avoid problems when sending stuff to oneself. Will make sure to clarify there.